src/app/components/SearchModal/filtering.ts

```diff
@@ -35,11 +35,11 @@ export function createTokenFilterFunction<T extends Token | TokenInfo>(search: s
     return lowerSearchParts.every(p => p.length === 0 || sParts.some(sp => sp.indexOf(p) >= 0));
   };
 
-  return ({ name, symbol, searchableTerms }: T): boolean =>
+  // consider whether the searchableTerms is useful
```

Comment:
I guess we can leave this out for now and revisit later when there are more tokens and some additional search logic would be desired. Token symbol and name is good enough. @parrot9design any strong preference here?

Reply:
Name and symbol should be enough for now. Does the current token list include any additional search logic that this PR has missed?

Reply:
The token list doesn't have the logic.